c-codes/LargeNumberMultiplication.c

```diff
@@ -0,0 +1,40 @@
+#include "stdio.h"
+#include "string.h"
+#define MAX 10000//结果最大位数，可以自行扩大·以扩大范围
```

Comment:
change to english please @ccm18454090438

Reply:
@UDLD  fxed, please review